template.yaml

```diff
@@ -115,14 +115,25 @@ Resources:
     Type: AWS::S3::Bucket
 
   DynamoSiteTable:
-    Type: AWS::Serverless::SimpleTable # if you want to define a more complex table, use AWS::DynamoDB::Table
+    Type: AWS::DynamoDB::Table
     Properties:
       TableName: !Sub "federalist-proxy-${StackEnv}"
-      PrimaryKey:
-        Name: Id
-        Type: String
-      Tags:
-          AppType: Serverless
+      AttributeDefinitions:
+        - AttributeName: Id
+          AttributeType: S
```

Comment:
the type is just `S`?

Reply:
yep .. the options are :
S -> String
N -> Number
B -> Binary